src/packages/@ncigdc/tableModels/utils.tsx

```diff
@@ -83,36 +96,55 @@ export const createDataCategoryColumns = ({
           >
             {_.sumBy(
               hits.edges,
-              x =>
+              (x: any) =>
                 findDataCategory(category.abbr, x.node.summary.data_categories)[
                   countKey
-                ],
+                ]
             ).toLocaleString()}
           </Link>
         </TdNum>
       ),
     })),
   ];
 };
-
-type TCreateSelectColumn = ({
-  idField: string,
-  headerRowSpan?: number,
-}) => {};
-export const createSelectColumn: TCreateSelectColumn = ({
+export interface IColumnProps<NoTH> {
+  name: string;
+  id: string;
+  sortable: boolean;
+  downloadable: boolean;
+  hidden: boolean;
+  field?: string;
+  subHeading?: boolean;
+  subHeadingIds?: NoTH extends true? string[] : undefined;
+  parent?: string;
+  th: (props: any) => JSX.Element;
+  td: NoTH extends true? undefined : (props: any) => JSX.Element;
+}
+export const createSelectColumn = ({
   idField,
   headerRowSpan,
-}) => {
+}: {
+  idField: string;
+  headerRowSpan?: number;
+}): IColumnProps<false> => {
   return {
     name: 'Select',
     id: 'select',
     sortable: false,
     downloadable: false,
     hidden: false,
-    th: ({ nodes, selectedIds, setSelectedIds }) => {
+    th: ({
+      nodes,
+      selectedIds,
+      setSelectedIds,
+    }: {
+      nodes: Array<IColumnProps<false>>;
+      selectedIds: string[];
+      setSelectedIds: (props: string[]) => void;
+    }) => {
       // NOTE: "nodes" is really "edges" in the graphql schema
-      const ids = nodes.map(node => node[idField] || node.node[idField]);
-      const allSelected = ids.every(id => selectedIds.includes(id));
+      const ids = nodes.map((node: any) => node[idField] || node.node[idField]);
```

Comment:
Same comment as above, you can see a similar solution around the CancerDistributionTable